src/System.Private.CoreLib/src/System/Threading/Thread.CoreCLR.cs

```diff
@@ -397,7 +397,7 @@ public ThreadPriority Priority
 #if FEATURE_COMINTEROP_APARTMENT_SUPPORT
             (ApartmentState)GetApartmentStateNative();
 #else // !FEATURE_COMINTEROP_APARTMENT_SUPPORT
-            ApartmentState.MTA;
+            ApartmentState.Unknown;
```

Comment:
Also, `TrySetApartmentStateUnchecked` should be `internal`

Reply:
Seems like it could be `private`?

Reply:
Agree